index.html

```diff
@@ -97,106 +97,31 @@ <h1 class="my_name">
-                </ul>
-              <button class="proj_view_btn" type="button">
-                See this Project <i class="fa fa-arrow-right"></i>
-              </button>
+         
+          <div class="project_list" id="list-2">
             </div>
+          
+         <div class="project_list" id="list-3">
             </div>
-
-            <div class="project_list" id="list3">
-              <img
-                class="proj_1st_image"
-                id="3work_img"
-                src="img/Work 3.png"
-                alt="3rd project image"
-              />
-              <div class="proj_list_container">
-                <h4 class="proj_name">Project name goes here</h4>
-                <ul class="lang_list">
-                  <li><a href="#">HTML/CSS</a></li> 
-                  <li><a href="#">Ruby on Rails</a></li>
-                  <li><a href="#">JavaScript</a></li>
-                </ul>
-                <button class="proj_view_btn" type="button">
-                  See this Project <i class="fa fa-arrow-right"></i>
-                </button>
-              </div>
             </div>
           </div>
             <div class="align_proj_4n5">
-             <div class="project_list" id="list4">
-              <img
-                class="proj_1st_image"
-                src="img/Work 4.png"
-                alt="4th project image"
-              />
-              <div class="proj_list_container">
-                <h4 class="proj_name">Project name goes here</h4>
-                <ul class="lang_list">
-                  <li><a href="#">HTML/CSS</a></li> 
-                  <li><a href="#">Ruby on Rails</a></li>
-                  <li><a href="#">JavaScript</a></li>
-                </ul>
-                <button class="proj_view_btn" type="button">
-                  See this Project <i class="fa fa-arrow-right"></i>
-                </button>
+             <div class="project_list" id="list-4">
               </div>
-            </div>
-          
+            <div></div>
             <div class="project_list" id="list-5">
-              <img
-                class="proj_1st_image"
-                src="img/Work 5.png"
-                alt="5th project image"/>
-              <div class="proj_list_container">
-                <h4 class="proj_name">Project name goes here</h4>
-                <ul class="lang_list">
-                  <li><a href="#">HTML/CSS</a></li> 
-                  <li><a href="#">Ruby on Rails</a></li>
-                  <li><a href="#">JavaScript</a></li>
-                </ul>
-                <button class="proj_view_btn" type="button">
-                  See this Project <i class="fa fa-arrow-right"></i>
-                </button>
-              </div>
             </div>
             </div>
-
-            <div class="final_proj">
+          <div class="final_proj">
               
             <div class="backg_fin"></div>
             <div class="project_list" id="list-6">
```

Comment:
- [ ] It is required to remove all the previous html code and create them dynamically using javascript. Kindly remove this part too. Take a look at the project requirement that stated this. :point_down: 

![Uploading project-req.png…]()